packages/@aws-cdk/aws-sns/test/subscription.test.ts

```diff
@@ -176,7 +176,7 @@ describe('Subscription', () => {
 
   });
 
-  test('throws with raw delivery for protocol other than http, https or sqs', () => {
+  test('throws with raw delivery for lambda protocol', () => {
```

Comment:
❓  Why did you make this change? While the text more accurately describes the specifics of the test, they lose the _intent_ behind the test.

Reply:
@njlynch , the choice was between changing the name to "protocol other than http, https, sqs, or firehose" (since firehose supports raw delivery) or changing it to "lambda protocol", which has the advantage of being more accurate. As it is, the test name is misleading - it doesn't cover email or sms which also don't support raw delivery. Better just to name it more transparently.